roles/foreman_repositories/tasks/redhat.yml

```diff
@@ -48,8 +72,14 @@
       register: disabled_repos
       changed_when: false
 
-    - name: Enable powertools
-      command: dnf config-manager --set-enabled powertools
+    - name: Enable PowerTools on CentOS
+      command: dnf config-manager --set-enabled PowerTools
```

Comment:
I am still seeing this as `powertools` in my environments.

Reply:
Hmm, interesting, I ran into errors as it was in CamelCase on CentOS8.3 system. Anyway, I'll stick it in a rescue block then :-)